libs/designer/src/lib/core/state/workflow/workflowSelectors.ts

```diff
@@ -135,6 +149,21 @@ export const useWorkflowNode = (actionId?: string) => {
   });
 };
 
+export const useWorkflowNodeIndex = (actionId: string, runAfters: string[]): number => {
```

Comment:
This should be more descriptive, explain what you mean by index, index in relation to what?

Reply:
`useRunAfterIndexBySource` or something along those lines